Common/CostFunctions/itkAdvancedImageToImageMetric.hxx

```diff
@@ -781,7 +781,7 @@ AdvancedImageToImageMetric<TFixedImage, TMovingImage>::GetValueThreaderCallback(
 
   temp->st_Metric->ThreadedGetValue(threadID);
 
-  return itk::ITK_THREAD_RETURN_DEFAULT_VALUE;
+  return ITK_THREAD_RETURN_DEFAULT_VALUE;
```

Comment:
This commit is easy for me to grasp. I don't like the unnecessary `itk::` in front of each ITK_THREAD_RETURN_DEFAULT_VALUE anyway. Shall I already cherry-pick and merge this one? (The other commits may take me some more time.)

Reply:
Feel free to cherry-pick / merge.

I need all the commits in this branch for a working build used by 
https://github.com/InsightSoftwareConsortium/ITKElastix/pull/227